source/_docs/lockr.md

```diff
@@ -0,0 +1,162 @@
+
+Lockr allows site administrators the ability to take control of how and where their sensitive keys are stored, thus improving the overall security of the site and allows it to meet specific regulatory and compliance requirements around key management. Lockr also separately stores development and production specific keys to create an additional layer of security between environments.
+
+## Why Lockr?
+
+By combining a simple to use developer interface with a managed scalable cloud key management system, Lockr allows applications of all sizes to meet industry standards for key management. Unlike other key managers, Lockr offers a "white glove service" that provides additional layers of security and system monitoring, no ongoing maintenance and continuous development for integrations with your favorite modules and plugins. Offsite API & encryption key management delivers best-practice security to help sites comply with HIPAA, FERPA, and FISMA among others. Development use of Lockr in the Dev, Test and Multi-Dev environments is always free.  
+
+## WordPress Installation
+
+Lockr is automatically configured to secure API keys for numerous 3rd party plugins for seamless integration and securing of your keys. 
+Visit our [Github page](https://github.com/lockr/lockr-patches/tree/wp) for a list of plugins that can be automatically patched.
+
+### Install through the Administration Interface
+
+The official [Lockr Plugin](https://wordpress.org/plugins/lockr) makes it a breeze to get started and quickly secure your keys. Install and activate the latest release through the WordPress dashboard or place it it in /wp-content/plugins directory and activate via the dashboard.
+
+1. Install through the WordPress plugin screen directly or upload the Lockr package to `/wp-content/plugins/` directory. 
+2. Activate Lockr through the “Plugins” section in your WordPress backend  (ex. `http://my-site.pantheonsite.io/wp-admin/plugins.php` )
+3. Visit the Lockr Configuration page  (ex. `http://my-site.pantheonsite.io/wp-admin/admin.php?page=lockr-site-config`)
+
+![WP Lockr Configuration example](/source/docs/assets/images/lockr_wpconfig.png)​
+
+4. Enter your email address and Lockr will take care of the rest! (note: For first time users, you will be receiving an email with instruction, if an existing Lockr users should use the email address they used to create their Lockr account. After submitting the request to register the site, you will be prompted for your Lockr password to confirm adding the site to your account.)
```

Comment:
<ol start="4">
<li>Enter your email address and click **Register Site**.
<div class="alert alert-info">
<h4>Note</h4>
First time Lockr users will receive an email with account information. Existing Lockr users should provide the email address associated with your Lockr account and authenticate when prompted. 
</div>
</li>